data/drak/drak outfits.txt

```diff
@@ -176,90 +324,11 @@ effect "antimatter dust"
 
 
 
-ship "Archon"
-	noun "entity"
-	sprite "ship/archon"
-		"frame rate" 0.87
-	thumbnail "thumbnail/archon"
-	"never disabled"
-	attributes
-		category "Heavy Warship"
-		"cost" 1000000000
-		"shields" 4000000
-		"hull" 1000000
-		"required crew" 1
-		"bunks" 1
-		"mass" 1000
-		"drag" 27
-		"heat dissipation" 40
-		"fuel capacity" 800
-		"cargo space" 0
-		"outfit space" 800
-		"weapon capacity" 600
-		"engine capacity" 200
-		"energy capacity" 10000
-		"energy generation" 500
-		"heat generation" 17
-		"shield generation" 100
-		"shield energy" 100
-		"hull repair rate" 100
-		"hull energy" 100
-		"ion protection" 3
-		"ion resistance" 1
-		"ramscoop" 100
-		weapon
-			"blast radius" 200
-			"shield damage" 20000
-			"hull damage" 10000
-			"hit force" 30000
-	outfits
-		"Drak Antimatter Cannon"
-		"Drak Anti-Missile Field" 2
-		"Drak Draining Field"
-		"Drak Distancer" 2
-		"Drak Turret" 3
-		
-		"Medium Graviton Thruster"
-		"Medium Graviton Steering"
-		"Hyperdrive"
-		"Jump Drive"
-	
-	engine -17 92
-	engine 17 92
-	gun 0 -74 "Drak Antimatter Cannon"
-	turret 0 -10 "Drak Anti-Missile Field"
-	turret 0 -10 "Drak Anti-Missile Field"
-	turret 0 -10 "Drak Draining Field"
-	turret -40 -13 "Drak Turret"
-	turret 40 -13 "Drak Turret"
-	turret 0 81 "Drak Turret"
-	turret -55 16 "Drak Distancer"
-	turret 55 16 "Drak Distancer"
-	explode "nuke explosion" 3
-	"final explode" "final explosion large"
-	description `The Archon's presence is chilling. The "ship" has an almost organic appearance, but your ship's scanners give you no discernible information on its construction. What exactly it is you do not know, but you can feel that it is powerful.`
-
-ship "Archon" "Archon B"
-	"never disabled"
-	sprite "ship/archon b"
-		"frame rate" 0.87
-
-ship "Archon" "Archon C"
-	"never disabled"
-	sprite "ship/archon c"
-		"frame rate" 0.87
-
-ship "Archon" "Archon (Cloaked)"
-	sprite "ship/archon c"
-		"frame rate" 0.87
-	"never disabled"
-	add attributes
-		"cloak" .04
-	turret "Drak Anti-Missile Field"
-	turret "Drak Anti-Missile Field"
-	turret "Drak Draining Field"
-	turret "Drak Turret"
-	turret "Drak Turret"
-	turret "Drak Turret"
-	turret "Drak Distancer"
-	turret "Drak Distancer"
+outfit "Drak Ammunition"
```

Comment:
There's no need for an ammunition outfit, let alone so much of it.
1) Ammunition has a chance to get dropped on death. With as much ammunition as you're giving Archons, they'll probably be spitting out a load of crates.
2) Ammunition eventually runs out. Now with how much ammo you've given Archons I think it would take over 12 days straight to run out of ammo, but it'd be better if we didn't have to give the Archon so much and it never ran out.

Instead of having an ammo outfit, you should be able to just give every "augmented" weapon `ammo "Jump Drive" 0`, or choose any other outfit the Archon has installed. Since the weapon requires ammo to fire it should still be affected by the frugal personality, and an ammo requirement of 0 means it never actually uses up the ammo.

Reply:
OK, made the change.

Reply:
Still need to scrub the capacities from the weapons.